columbo/_interaction.py

```diff
@@ -556,7 +619,8 @@ def get_answers(
 
     for interaction in interactions:
         if isinstance(interaction, (Echo, Acknowledge)):
-            interaction.display(result)
+            if interaction.should_ask(result):
```

Comment:
Since, both cases now call `should_ask()`, that can be lifted out and called once before doing the `isinstance()` checks.

Reply:
I can't do that without failing a test:

```
FAILED tests/interaction_test.py::test_get_answers__unknown_interaction_provided - AttributeError: 'int' object has no attribute 'should_ask'
```

Reply:
Oh right, I forgot about that. It was a minor thing, so we can leave it as is.